src/components/context/UserContextProvider.tsx

```diff
@@ -0,0 +1,45 @@
+// @ts-ignore

```

Comment:
What's being ignored? I'd rather we didn't ignore errors and if we do then provide a comment as to why